### PR TITLE
Adds Contribution Synergy to "first-repo" Repo

### PR DESCRIPTION
**Pull request by Schachte**
Updating contributions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
 # first-repo
+
+# Contributors:
+@schachte
+@rhett-raleigh
```
Comment (bdeining) [inline README.md:5]: please add @bdeining

**Review by bdeining — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Schachte) [reply README.md:5]: Added!

**Review by bdeining — APPROVED** [review]

**Comment by bdeining** [comment]
@rhett-raleigh please merge

**Comment by Schachte** [comment]
@rhett-raleigh Checking in on status

**Pull request closed without merging**